src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -48,5 +54,18 @@ export default {
 			return this.userInfo?.firstName ?? '';
 		}
 	},
+	methods: {
+		async getRecommendedLoans() {
+			const { loans } = await runLoansQuery(
+				this.apollo,
+				{ sortBy: 'personalized', pageLimit: 6 },
+				FLSS_ORIGIN_NOT_SPECIFIED
```

Comment:
Should this be a more specific origin?

Reply:
yeah, waiting for DS team answer, I can change it in a next ticket.